config/environments/production.rb

```diff
@@ -101,7 +101,7 @@
 
   # DEV uses the RedisCloud Heroku Add-On which comes with the predefined env variable REDISCLOUD_URL
   redis_url = ENV["REDISCLOUD_URL"]
-  redis_url ||= ApplicationConfig["REDIS_URL"]
+  redis_url ||= ENV["REDIS_URL"]
```

Comment:
Without this I always got a `uninitialized constant ApplicationConfig (NameError)` on Travis and locally. I think it makes sense to keep this consistent with line 103 anyway.